snuba/datasets/entities/metrics.py

```diff
@@ -161,6 +171,17 @@ def __init__(self) -> None:
         )
 
 
+class OrgMetricsCountersEntity(MetricsEntity):
+    def __init__(self) -> None:
+        super().__init__(
+            writable_storage_key=None,
+            readable_storage_key=StorageKey.METRICS_COUNTERS,
+            value_schema=[Column("value", AggregateFunction("sum", [Float(64)]))],
```

Comment:
I don't think this is meant to expose all the fields.
See how this was done on sessions
https://github.com/getsentry/snuba/blob/master/snuba/datasets/entities/sessions.py#L305-L312

That only included org, project and time exactly to ensure this entity would not be used for anything other than returning a list of orgs and projects as it does not have the safeguards in place to prevent a query across organizations done by mistake.

I think we should do the same and only expose the same fields: org, project, timestamp (and granularity otherwise you cannot run the query).

We can consider tags if needed, not sure how the query is meant to look like, but please do not expose the aggregate values otherwise this could be used for scenarios other than your specific one.